CONTRIBUTING.md

```diff
@@ -78,27 +78,31 @@ Boulder attempts to write that struct to a table that doesn't yet have the
 corresponding column (case 1), gorp wil fail with
 `Insert failed table posts has no column named Foo`.
 There are examples of such models in sa/model.go, along with code to
-turn a model into a `struct` used internally. An example of a flag-gated
-migration, adding a new `IsWizard` field to Person:
+turn a model into a `struct` used internally. 
+
+An example of a flag-gated migration, adding a new `IsWizard` field to Person
+controlled by a `allowWizards` flag:
 
 ```
 struct Person {
   HatSize  int
   IsWizard bool // Added!
 }
 
+var allowWizards bool // Added!
+
 struct personModelv1 {
   HatSize int
 }
 
 // Added!
 struct personModelv2 {
```

Comment:
Should we have a example from removing a field since it'll be a rather different process?